CriteoPublisherSdk/Tests/UnitTests/Network/CR_NetworkManagerTests.m

```diff
@@ -173,14 +173,14 @@ - (void)testNetworkManagerPostCall {
 }
 
 - (void)testNetworkManagerFailedSerialization {
-    NSURL *url;
-    NSData *body;
-    [self.networkManager postToUrl:url
-                              body:body
-                   responseHandler:^(NSData * _Nullable data, NSError * _Nullable error) {
-        XCTAssertNotNil(error);
-        XCTAssertNil(data);
-    }];
+  NSURL *url;
+  NSData *body;
+  [self.networkManager postToUrl:url
+                            body:body
+                 responseHandler:^(NSData *_Nullable data, NSError *_Nullable error) {
+                   XCTAssertNotNil(error);
```

Comment:
Why not fixing the initial commit by amending it rather than adding many fixup commits which could add noise when digging in the repository ? Also it make things harder to be reverted as you'll need to revert many commit at once.